compiler/transforms/convert_linalg_to_acc.py

```diff
@@ -0,0 +1,237 @@
+from collections.abc import Sequence
+from dataclasses import dataclass, field
+
+from xdsl.dialects import arith, builtin, func, linalg, memref, scf
+from xdsl.ir import Block, MLContext, Operation, Region, SSAValue
+from xdsl.passes import ModulePass
+from xdsl.pattern_rewriter import (
+    PatternRewriter,
+    PatternRewriteWalker,
+    RewritePattern,
+    op_type_rewrite_pattern,
+)
+
+from compiler.dialects import acc
+
+
+class AcceleratorConfig:
```

Comment:
This feels like it does not really belong here? `AcceleratorConfig` should also be an abstract class I feel, now it is confusing that `snax_hwpe_mult` is the name of the class

Reply:
Yes, this was an abstract class once upon a time, but then we wanted to be a bit more concrete. Never made it more general again.